examples/CSharp/Tutorial 1.5 - Transformations/Shader.vert

```diff
@@ -0,0 +1,13 @@
+#version 330 core
```

Comment:
Another thing: I hate to sound like my computer science teacher yelling at me for not commenting a hello world program.....

.....but perhaps you should put some comments in the shaders?? :P

Reply:
im thinking about not doing this for the hello quad as that would leave us trying to explain more code than necessary for your first quad.

Reply:
Okay, but to my knowledge none of the shaders are commented and I think some explaination of what's happening, however brief, should be put in via comments. I'm not after much, just a brief sentence or two explaining what's going on)